RELEASING.md

```diff
@@ -8,45 +8,47 @@ The Google Cloud Ruby Client project uses [semantic versioning](http://semver.or
+    $ git status
+    $ git pull <remote> master --rebase
     ```
 
-1. Install the gem locally.
+1. Build the gem locally. (Depending on your environment, you may need to `bundle exec` to rake commands; this will be shown.)
 
     ```sh
-    $ rake install
+    $ cd <gem>
+    $ bundle exec rake build
     ```
 
-1. Using IRB (not `rake console`!), manually test the gem that you installed in the previous step.
-
-1. Review the changes since the last release.
+1. Install the gem locally.
 
     ```sh
-    $ bundle exec rake changes[<gem>]
+    $ bundle exec rake install
     ```
 
-1. Review the commits in the changes output, making notes of significant changes. (For examples of what a significant change is, browse the changes in the gem's `CHANGELOG.md`
+1. Using IRB (not `rake console`!), manually test the gem that you installed in the previous step.
 
-1. If you haven't already, switch to the master branch, ensure that you have no changes, and pull from origin.
+1. Return to the root directory of the project, and review the changes since the last release.
 
     ```sh
-    $ git checkout master
-    $ git status
-    $ git pull <remote> master --rebase
+    $ cd ..
+    $ bundle exec rake changes[<gem>]
     ```
 
+1. Review the commits in the changes output, making notes of significant changes. (For examples of what a significant change is, browse the changes in the gem's `CHANGELOG.md`.
+
 1. Edit the gem's `CHANGELOG.md`. Using your notes from the previous step, write bullet-point lists of the major and minor changes. You can also add examples, fixes, thank yous, and anything else helpful or relevant. See google-cloud-node [v0.18.0](https://github.com/GoogleCloudPlatform/google-cloud-node/releases/tag/v0.18.0) for an example with all the bells and whistles.
 
 1. Edit the gem's `version.rb` file, changing the value of `VERSION` to your new version number.
 
 1. Edit the gem's entry in `docs/manifest.json`, adding your new version number to the head of the list, and moving `"master"` to be just below it.
 
-1. If your version change is greater than the [semver](http://semver.org/) patch version, edit the requirement for the gem in `google-cloud/google-cloud.gemspec` and `stackdriver/stackdriver.gemspec`, replacing the old version number for the gem with your new version number.
+1. If the package is `< 1.0.0` and your version change is greater than the [semver](http://semver.org/) patch version, or if the package is `>= 1.0.0` and your version change increments the [semver](http://semver.org/) major version, edit the requirement for the gem in `google-cloud/google-cloud.gemspec` and `stackdriver/stackdriver.gemspec` (if the package is a dependency of stackdriver) , replacing the old version number for the gem with your new version number.
 
-1. Test all the version dependencies are correct.
+1. In the root directory of the project, test that all the version dependencies are correct.
 
     ```sh
     $ bundle update
```

Comment:
Below (line 64): I was curious about this last `git pull`. If commits have been added between when you last `pull`ed above and here, ISTM you shouldn't include them in the tag, since none of the verification steps you just did apply to the new commits.

Reply:
@blowmage This pre-dates me. What's your take on this step?

Reply:
Something like `bundle exec rake test` will do this. If it runs then you can be fairly confident you don't have any incorrect dependencies.

Reply:
@blowmage Right, you've already effectively done that in a previous step (`bundle exec rake ci[yes]`). My point is that if you run that, and then you pull new commits afterward, you lose the verification you just did.